tycho-extras/tycho-pomless/src/main/java/org/eclipse/tycho/pomless/TychoAggregatorMapping.java

```diff
@@ -110,11 +112,6 @@ protected void initModel(Model model, Reader artifactReader, File artifactFile)
         }
     }
 
-    @Override
-    public float getPriority() {
-        return -10f;
```

Comment:
Anything wrong with negative priority? The idea was that this mapping is asked only as a last resort if nothing else matches.

Reply:
Nope not at all. It was just with the initially suggested orders the XMLMapping became de-facto unused for all PDE Project-types and I therefore moved it above the XMLMapping.
With the now implemented solution it must have a lower priority than the XMLMapping so I reverted to -10.